src/components/Icon/icons/ArrowDown.tsx

```diff
@@ -0,0 +1,13 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
```

Comment:
did you manually add these headers? if so we should also automate this in another sub task of `yarn svg`

Reply:
(but i had not found a good solution yet either)